### PR TITLE
feat: Prisma relational schema 

### PR DESCRIPTION
**Pull request by arya2004**
#7

### CODE REVIEW

File: src/app/api/getPosts/route.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+interface Post {
+    title: string;
+    content: string | null;
+    published: boolean;
+}
+
+
+export async function GET() {
+    const data = await prisma.post.findMany();
+    return Response.json(
```
Comment (Soham1803) [inline src/app/api/getPosts/route.ts:17]: Use Nextresponse instead of Response everywhere.

File: .env [hunk 2]
```diff
@@ -4,4 +4,4 @@
 # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="mongodb+srv://mlsc:mlsc1234@cluster0.wiejgzj.mongodb.net/mlsc-website?retryWrites=true&w=majority"
\ No newline at end of file
+DATABASE_URL="postgresql://postgres:postgres123@localhost:5432/MLSC?schema=public"
```
Comment (Soham1803) [inline .env:7]: Add this file in gitignore below .env.local.*

File: prisma/schema.prisma [hunk 3]
```diff
@@ -10,18 +10,63 @@ datasource db {
...
-  //posts Post[]
+  id      Int      @id @default(autoincrement())
+  fullName    String?
+  domain String
+  position String
+  year Int
+  xLink String
+  linkedinLink String
+  githubLink String
+  aboutMe String
```
Comment (Soham1803) [inline prisma/schema.prisma:23]: Make aboutMe optional using ?

**Comment by arya2004** [comment]
fixed issues in latest comit

**Review by Soham1803 — APPROVED** [review]
LGTM!

**Pull request merged**